common/ta_errors.h

```diff
@@ -179,6 +180,28 @@ typedef enum {
   SC_UTILS_TIMER_ERROR = 0x02 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
   /**< Errors occurred in timer function */
   SC_UTILS_TIMER_EXPIRED = 0x03 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  /**< Failed to send message */
+  SC_UTILS_HTTPS_SEND_ERROR = 0x04 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  /**< HTTPS module initialize error */
+  SC_UTILS_HTTPS_INIT_ERROR = 0x05 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  /**< HTTPS X509 certificate parse error */
+  SC_UTILS_HTTPS_X509_ERROR = 0x06 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  /**< HTTPS initial connection error */
+  SC_UTILS_HTTPS_CONN_ERROR = 0x07 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  /**< HTTPS setting SSL config error */
+  SC_UTILS_HTTPS_SSL_ERROR = 0x08 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  /**< HTTPS response error */
+  SC_UTILS_HTTPS_RESPONSE_ERROR = 0x09 | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
+  /**< Error occured when serialize message */
+  SC_UTILS_TEXT_SERIALIZE = 0x0A | SC_MODULE_UTILS | SC_SEVERITY_FATAL,
```

Comment:
Both `SC_UTILS_TEXT_SERIALIZE` and `SC_UTILS_TEXT_DESERIALIZE` should be revised for better understanding.